qtrobot:
    # Publish all joint states -----------------------------------
    joint_state_controller:
      type: joint_state_controller/JointStateController
      publish_rate: 50

    # Position Controllers ---------------------------------------
    yaw_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: HeadYaw
      pid: {p: 1.0, i: 1.0, d: 0.0}
    pitch_joint_position_controller:
      type: effort_controllers/JointPositionController
      joint: HeadPitch
      pid: {p: 1.0, i: 1.0, d: 0.0}
    # To add more just add them here as the first one